src/components/Button/Button.stories.js

```diff
@@ -2,14 +2,15 @@ import Button from './Button'
 
 // eslint-disable-next-line import/no-anonymous-default-export
 export default {
-  title: 'Capstone Project/Button',
+  title: 'Capstone Project/2-Buttons',
```

Comment:
Even though you have multiple buttons you render here, this is the story just for the `Button` component so therefore the name for this is `Button`.
```suggestion
  title: 'Capstone Project/Button',
```